### ISSUE TITLE
Cannot be installed with wp-cli 2.2.0 out of the box

### ISSUE THREAD

**Issue by ratzrattillo** [diggy/polylang-cli#118]
Hey there,

i would like to try out your very promising package, as the developers of polylang obviously do not yet have any interest in providing a CLI interface.

However when trying to install the pakcage, i get the following errors:
```
wp package install git@github.com:diggy/polylang-cli.git

Installing package diggy/polylang-cli (dev-master)
Updating /home/www-data/.wp-cli/packages/composer.json to require the package...
Registering git@github.com:diggy/polylang-cli.git as a VCS repository...
Using Composer to install the package...
 
Loading composer repositories with package information
Updating dependencies
Resolving dependencies through SAT

Dependency resolution completed in 0.016 seconds
Your requirements could not be resolved to an installable set of packages.
Problem 1
    - Conclusion: remove wp-cli/wp-cli 2.2.0
    - diggy/polylang-cli dev-master requires wp-cli/wp-cli ~1.5.0 -> satisfiable by wp-cli/wp-cli[v1.5.0, v1.5.1].
    - diggy/polylang-cli dev-master requires wp-cli/wp-cli ~1.5.0 -> satisfiable by wp-cli/wp-cli[v1.5.0, v1.5.1].
    - Can only install one of: wp-cli/wp-cli[v2.2.0, v1.5.0].
    - Can only install one of: wp-cli/wp-cli[v2.2.0, v1.5.1].
    - Installation request for wp-cli/wp-cli 2.2.0 -> satisfiable by wp-cli/wp-cli[2.2.0].
    - Installation request for diggy/polylang-cli dev-master -> satisfiable by diggy/polylang-cli[dev-master].
Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements ablocking the update you have to resolve those problems.
 
Error: Package installation failed (Composer return code 2).
Reverted composer.json.
```

Do you know how to fix this, so the future users can enjoy your tool even with the newest version of wp-cli?
Did you ever think about submitting your polylang-cli project to the official polylang project? I think this would be really useful and maybe they would take over the maintenance for you also .:)

Greetings!

**Comment by szepeviktor** [comment]
Of course, see https://github.com/diggy/polylang-cli/blob/master/composer.json#L21

This project may need to wake up.

@diggy What do you think?

**Comment by diggy** [comment]
@ratzrattillo @szepeviktor I've been out of the WP ecosystem for a while, but it looks like I might be circling back over the course of the next few months. In the meantime, happy to accept pull requests!

**Comment by ratzrattillo** [comment]
@szepeviktor @diggy 
Thanks for your great help. Its working now perfectly with wp-cli 2.2.0.
I would love to see you return to WP and push your project further, as it is really useful!

**Comment by fredleeflang** [comment]
Hmm, I don't really understand @szepeviktor solution. In any case, I'm seeing the same error message when using 2.4.0. How can I fix this?

**Comment by fredleeflang** [comment]
@diggy could you perhaps elaborate on @szepeviktor solution?

**Comment by szepeviktor** [comment]
> This project may need to wake up.

I've wrote only this, no solution.

> Thanks for your great help. Its working now perfectly with wp-cli 2.2.0.

I totally don't understand it.
AFAIK polylang-cli works only with WP-CLI v1

**Comment by fredleeflang** [comment]
gotcha @szepeviktor I guess @ratzrattillo had us both confused :)
Listen, I don't see many commits in this package. As this is one of the first packages for WP CLI that I even look into, maybe I can volunteer to help a bit. I'll clone from git and try to resolve the issue. When I succeed I'll offer a fix.

**Comment by szepeviktor** [comment]
Just have @diggy merge it!

**Comment by szepeviktor** [comment]
https://packagist.org/packages/szepeviktor/phpstan-wordpress may help you find and fix hidden bugs

**Comment by vicksbr** [comment]
Have you guys found a solution/workaround?

**Comment by dejwsz** [comment]
So is it possible to install it and use with WP CLI in version 2.X.X ? Anyone succeeded with it?

**Comment by robertcurcio** [comment]
Ping @diggy
